.travis.yml

```diff
@@ -21,6 +21,7 @@ cache:
         - $HOME/.composer/cache
 
 before_script:
+    - sudo apt-get install php-zip
```

Comment:
Can you try with `sudo apt-get -q update && sudo apt-get -y install php-zip`? I think the error is due to apt database missing without `update`.